cli/src/wallet_cli/mod.rs

```diff
@@ -145,12 +142,18 @@ pub enum WalletCommand {
     MintNft {
         /// Address to send the NFT to, e.g. rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3.
         address: Option<Bech32Address>,
+        /// Immutable metadata key, e.g. --immutable-metadata-key data.
+        #[arg(long, group = "immutable_metadata", default_value = "data")]
```

Comment:
I don't think it should be part of the group? Group means only one of them I think

Reply:
A group can also be configured, but yeah easier to just not have it in there